Configurations/unix-Makefile.tmpl

```diff
@@ -520,10 +520,12 @@ clean: libclean
 	$(RM) -r test/test-runs
 	$(RM) openssl.pc libcrypto.pc libssl.pc
 	-$(RM) `find . -type l \! -name '.*' -print`
+	$(RM) pod2htmd.tmp
```

Comment:
By the way, thought this is not an issue in your PR, I am rather uncomfortable with the line above:
```
$(RM) `find . -type l \! -name '.*' -print`
```
It is blatantly unsafe when any names in the output have whitespace.  If we can't rely on a portable 'find ... -delete' option, then we have to do the removal via a Perl script, or something similarly robust to unexpected file names.

Reply:
Damnit this perl method would be far more safe then

Reply:
Do we still have random symlinks?  I personally would *much* prefer to remove known files than that kind of broad removal, so if it was up to me, I'd remove that line, and then find out what files rename and remove them each explicitly.

Reply:
In versions before 1.1.0, we did symlink headers quite a bit.  That line was much more justifiable then than now.